src/lib/trajectory/parse.ts

```
@@ -1,828 +1,499 @@
-    (content instanceof ArrayBuffer && content.byteLength === 0)
-  ) return Boolean(has_hdf5_extension)
+  // If no content or empty, return based on extension
+  if (!content || (content instanceof ArrayBuffer && content.byteLength === 0)) {
+    return Boolean(has_hdf5_extension)
+  }
 
   // Check if content is binary (HDF5 files are binary)
-  if (typeof content === `string`) {
-    return false // HDF5 files should not be parsed as text
-  }
+  if (typeof content === `string`) return false
```

Biome (1.9.4)

[error] Invalid `typeof` comparison value: this expression is not a string literal

not a string literal

(lint/suspicious/useValidTypeof)